### PR TITLE
Update bookmark list when archiving Questions in Collection page

### PR DESCRIPTION
**Pull request by gusaiani**
### How to Test

1. Create, save and bookmark a Question.
2. Go to http://localhost:3000/collection/root

You should see your question in the bookmarks list.

3. From the main collection list of items, open the ellipsis menu and click on `Archive`.

Your question should no longer be displayed (1) in the collection’s item list and (2) in the bookmarks list on the left.

—————

🎗️ I'll do one PR for each type of entity, and one for Models. Small PRs.

### CODE REVIEW

File: frontend/src/metabase/entities/bookmarks.js [hunk 1]
```diff
@@ -23,6 +25,18 @@ const Bookmarks = createEntity({
...
+    if (type === QUESTION_UPDATE) {
+      if (payload?.object?.archived === true) {
+        state[`card-${payload?.object?.id}`] = undefined;
+      }
+
+      return state;
+    }
+
+    return state;
+  },
```
[the remark this answers is not in the document]
Comment (gusaiani) [reply frontend/src/metabase/entities/bookmarks.js:39]: @kulyk, thank you very much.

File: frontend/src/metabase/entities/bookmarks.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { createEntity } from "metabase/lib/entities";
 import Collection from "metabase/entities/collections";
 import Dashboard from "metabase/entities/dashboards";
-import Question from "metabase/entities/questions";
+import Questions from "metabase/entities/questions";
```
Comment (kulyk) [inline frontend/src/metabase/entities/bookmarks.js:4]: nip: if you're renaming it to `Questions` (plural), should `Collection` and `Dashboard` be plural too?

**Review by kulyk — APPROVED** [review]
Nice!

File: frontend/src/metabase/entities/bookmarks.js (hunk 2 above)
Comment (gusaiani) [reply frontend/src/metabase/entities/bookmarks.js:4]: Yup, will apply one standard to these in follow up PRs.

**Pull request merged**